docs/learning-path.md

```diff
@@ -0,0 +1,88 @@
+---
+description: A collection of quick starts, courses and tutorials for n8n biginners to get started with n8n. 
+contentType: overview
+---
+Welcome to the n8n Learning Path! 
+
+If you're new to n8n, this guide is a suggested learning path to get started with n8n through tutorials, courses, and step-by-step guides. 
+
+/// note | Course duration and difficulty
+:material-clock-fast: : Quick tutorials
+:material-school: : Beginner friendly
+:material-school-outline: : Requries technical knowledge
+:fontawesome-solid-robot: Topics related to AI
+///
+
+## Try it out
+
+Start with the quickstart guides that will give you a fast taster of n8n and help you get up and running with building workflows. 
+
+<div class="grid cards" markdown>
+
+-   :material-clock-fast: __[Very quick quickstart :material-arrow-right-thick:](/try-it-out/quickstart/)__
+
+    Duration: **10 minutes**
```

Comment:
Do not add times. We're only going to annoy the people who take longer. And you really can't predict the time it will take. In particular, I know people who spend a day on one course level - but would be worries if the AI tutorial took an hour.